### ISSUE TITLE
error if gadgetron is not running 

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#28]
At present, if gadgetron is not running we get a difficult error message. in Python I get
```
??? 'Connection to Gadgetron failed' exception thrown at line 130 of /home/stir/devel/SIRF/src/xGadgetron/cGadgetron/xgadgetron.cpp
```
This might be related to reporting of exceptions as in #26. 

Can we make this error message clearer?
One suggestions is to add a simple test in setup_MR.

**Comment by evgueni-ovtchinnikov** [comment]
There can be at least three reasons for this error message:

1.       Gadgetron is not running.

2.       Gadgetron is running on VM, but port 9002 has not been forwarded.

3.       On Windows, I experienced frequent connection failures apparently caused by Windows’ interference. After some experimenting, I settled on using 5 connection attempts, and it has worked ok so far, but I cannot guarantee that this will always work.
Listing all these reasons in the error message is hardly reasonable. Perhaps we should mention them where we are going to describe how to use Gadgetron?
